### DISCUSSION TITLE
멀티룸 관련 질문

### DISCUSSION THREAD

**Discussion by hipsc159** [naverz/zepeto-studio-kor#325] [5. Q&A]
![Screenshot_1](https://user-images.githubusercontent.com/68143858/152622635-efec75bb-5bdf-4175-8d93-b7e13a90b948.png)

가이드에서  ZepetoWorldMultiplay 컴포넌트 추가하라고 하는데, 이 스크립트가 World.multiplay 폴더 안에 없네요.

스스로 직접 구현해야 하는 건가요? 아직 초보라서 뭐가 뭔지 잘 모르겠네요.. 하하

**Comment by x4cf** [comment]
제페토 월드 멀티플레이 스크립트는 프로젝트를 만들고 월드.멀티플레이를 만든다음 조금 늦게 활성화가 되더라구요. 차한잔 마시며 계속 껏다 켰다 하다보면 나타날거에요 XD

**Reply by hipsc159** [reply]
네 감사합니다!
